engine/executor/server.go

```diff
@@ -282,20 +281,14 @@ func (s *Server) Stop() {
 	}
 
 	if s.metastores.IsInitialized() {
-		etcdCli := s.metastores.ServiceDiscoveryStore()
-		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
-		defer cancel()
-		_, err := etcdCli.Delete(ctx, s.info.EtcdKey())
-		if err != nil {
-			log.L().Warn("failed to delete executor info", zap.Error(err))
-		}
```

Comment:
Do we need any operations equivalent to what you have deleted in order to remove the executor's information from the database as soon as possible?

Reply:
Yes. I will add a unregister method on master later. (Maybe in the next PR)